### ISSUE TITLE
Unhandled error when using document field in Admin UI

### ISSUE THREAD

**Issue by flexdinesh** [keystonejs/keystone#7800]
The document field often errors out and blocks the page. 

`Error: Cannot resolve a Slate point from DOM point: [object Text],17`

Most likely related to slate.

<img width="960" alt="Screen Shot 2022-08-11 at 12 01 47 pm" src="https://user-images.githubusercontent.com/5777880/184798031-f04a9c70-11c8-4a28-b1e4-38e54878ed99.png">

```
@keystone-6/fields-document: 4.0.1
Node version: v16.15.0
Browser: Chrome Version 104.0.5112.79 (Official Build) (arm64)
```

**Comment by willemmulder** [comment]
Yes, this has been reported before here: https://github.com/keystonejs/keystone/issues/7415
Great if it can be fixed 👍

**Comment by willemmulder** [comment]
Any update on this? This bug makes the document field pretty much unusable, which is too bad for such a nice feature.

**Comment by mauseoluwasegun** [comment]
### **Possible [Solutions:**](
before
![Image](https://github.com/user-attachments/assets/0c12b9af-82bc-456e-b2ad-c169146594ad)

**1.Try installing the missing slate-dom package if it's available**
      npm install slate-dom
      # or
      yarn add slate-dom
**2. Ensure that you're using compatible versions of all related packages:**
   npm list slate-react @keystone-6/fields-     document
# or
   yarn why slate-react
**3. In your package.json, you might need to add a resolution to force using a specific version of slate-react that doesn't depend on the separated slate-dom package:**
"resolutions": {
  "slate-react": "^0.65.0"  // Use a version before the split
}

**4. Then run:**
 npm install or yarn 

to update your dependencies.
5. npm run dev or yarn dev

issue is fixed

using the above command to fixed the issue:
![Image](https://github.com/user-attachments/assets/22aa6b06-47d2-4ef5-b0ba-7a25172b33d5)

After:

![Image](https://github.com/user-attachments/assets/01a0f735-9b06-4f93-a85c-ef47ce9ee433)

**Comment by dcousens** [comment]
Marking this as a duplicate of https://github.com/keystonejs/keystone/issues/7415
